src/stlink-lib/flash_loader.h

```diff
@@ -12,16 +12,10 @@
 
 #include <stlink.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif
 
```

Comment:
```suggestion
```

Reply:
Addressed in commit da3d9e3f1e1a6d116df5830cf506f30ff56196b3.